src/modules/aws_ec2/mappers/registered_instance.ts

```diff
@@ -136,20 +139,21 @@ export class RegisteredInstanceMapper extends MapperBase<RegisteredInstance> {
     create: (es: RegisteredInstance[], ctx: Context) => ctx.orm.save(RegisteredInstance, es),
     update: (es: RegisteredInstance[], ctx: Context) => ctx.orm.save(RegisteredInstance, es),
     delete: (es: RegisteredInstance[], ctx: Context) => ctx.orm.remove(RegisteredInstance, es),
-    read: async (ctx: Context, instAndTgArnAndPort?: string) => {
-      const opts = instAndTgArnAndPort
+    read: async (ctx: Context, id?: string) => {
+      const opts = id
```

Comment:
when calling explicitly `db.read` have you seen the result? Is it loading the instance object? or just returning the region and the port? I think this query misses the `relations: ['instance', 'targetGroup']` at the same level as the `where` prop

Reply:
So on the `entity` those columns are set to `eager: true` and all of the other tests directly interacting with the registered instances seems to work fine.